Resources/Maps/_NF/Shuttles/gazpacho.yml

```diff
@@ -0,0 +1,5718 @@
+
+
+        [head=3]Pre-Flight Checklist:[/head]
+          ◗ Ensure the generator is running. 25kW should cover the ship as delivered
+          ◗ Configure the air alarms from within the atmos observation area. The starboard chamber leads to breathable air tanks, the port side chamber is to be used for volatile gases.
+          ◗ The atmosphere in the living quarters may require manual refilling after several hours of continuous use. This can be done using the filled cans on the exterior platform.
+
+        ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
+
+
+        [head=3]General advice:[/head]
+
+          ◗ While considered "technically safe", advise all crew against being on the exterior walkways before commencing FTL-procedures. 
+
+          ◗ In the event of cataclysmic hellfires, or other undesirable conditions in one or both of the atmospheric holding chambers, please use the provdided remotes to vent out all gases and safely reset the environment for future use.
+
+          ◗ While tap water is available, it is advised that for reasons of cleanliness and morale all crew drink a beverage known as 'Grog' instead.
+
+          ◗ In case of emergencies or maintenance the shutters located in the living area and medbay can be used to access the thrusters from the interior of the ship. 
+
+          ◗ Do not under any circumstances eat any pills found aboard the vessle.
+        ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
+
+
+        By purchasing this ship you have signed an NDA with Imava Industries and agree to the following conditions:
+
+          ◗ [bolditalic]II LLC. can not be held responsible for any malfunctions, damages or fines caused by operator action or ship customization.[/bolditalic]
+          ◗ [bolditalic]II LLC. is not liable for any injuries or deaths resulting from crew incompetence.[/bolditalic]
+          ◗ [bolditalic]Microwaving the C.R.A.B™ voids your warranty.[/bolditalic]
+    - type: Forensics
+      residues: []
+      dnas: []
+      fibers: []
+      fingerprints:
+      - 670FB1B1B92C9A5E8797A1F1234A942C
+- proto: PaperCargoInvoice
+  entities:
+  - uid: 517
+    components:
+    - type: Transform
+      pos: -3.9347324,5.555607
+      parent: 1
+- proto: PaperOffice
+  entities:
+  - uid: 518
+    components:
+    - type: Transform
+      pos: -3.824051,5.3343678
+      parent: 1
+- proto: PartRodMetal
+  entities:
+  - uid: 273
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 3.240972,-3.2956343
+      parent: 1
+  - uid: 457
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 3.5730186,-3.2956343
+      parent: 1
+- proto: PlaqueAtmos
+  entities:
+  - uid: 511
+    components:
+    - type: Transform
+      pos: -0.5,4.5
+      parent: 1
+- proto: PlasticFlapsAirtightClear
+  entities:
+  - uid: 385
+    components:
+    - type: Transform
+      pos: -0.5,-6.5
+      parent: 1
+- proto: PortableGeneratorSuperPacmanShuttle
```

Comment:
Future changes will make this spread rads in the ship.

Reply:
YOOOOOOOO

Reply:
On this, if this is the case Im just going with my original power source plan of Damaged RTG. The fuel source would fit since it needs Gaseous Plasma in Radiation Collectors to produce power. The walls will be re-structured to fit this and become radiation-proof.

Reply:
Resolving cause outdated.